src/components/create-vm-dialog/pxe-helpers.js

```diff
@@ -136,7 +136,7 @@ export function getPXEInitialNetworkSource(nodeDevices, virtualNetworks) {
     );
 
     if (netNodeDevices.length > 0)
-        return cockpit.format('type=direct,source=$0', netNodeDevices[0].capability.interface);
+        return cockpit.format('type=direct,source=$0,source.mode=bridge', netNodeDevices[0].capability.interface);
```

Comment:
Hey. From what I understand, this specifies a source of the VM, meaning from where the VM will get installed (in this case, from network). This however doesn't specify virtual network interface which will persist as part of VM configuration once the VM gets installed right?
And from what I understand, main problem with using default VEPA mode, as according to https://bugzilla.redhat.com/show_bug.cgi?id=2095967, was that VMs with VEPA network interfaces could not talk to each other.
But since here we only specify source, and not network interface which may be used to communicate with other VMs, it's not a problem that we use VEPA here right?
It's just unless somebody complains that is is broken, I'm afraid to change something that is working. I'm afraid to change something which has been working without issues for years (i.e. if it works, don't touch it)
Or is there any other reason why using

Reply:
Nevermind, I just looked how we call virt-install and the the code from where gets used as parameters for "--pxe --network", which are then used as vNIC of the newly created VM. So retracting my review